### ISSUE TITLE
Error: lazy-load database is corrupt

### ISSUE THREAD

**Issue by ajabadi** [docker/build-push-action#74]
Hi,

Trying to build and push a successfully checked R package using GitHub Actions (https://github.com/ajabadi/BIRSBIO2020.scNMTseq.PLS/runs/875280609), I get the error:

```
2020-07-15T22:49:50.2499291Z [0m[91mError in get0(oNam, envir = ns) : 
2020-07-15T22:49:50.2499843Z   lazy-load dat
[12_Run dockerbuild-push-action@v1.txt](https://github.com/docker/build-push-action/files/4929435/12_Run.dockerbuild-push-action%40v1.txt)
abase '/usr/local/lib/R/site-library/pkgbuild/R/pkgbuild.rdb' is corrupt
```
 I'm attaching the log file. 
[Uploading 12_Run dockerbuild-push-action@v1.txt…]()
Not sure what the cause could be.

TIA

Al

**Comment by tonistiigi** [comment]
This looks like an issue in the application you are building, not in build action.

**Comment by ajabadi** [comment]
You're right. I managed to fix it for an R package container using [this setup](https://github.com/ajabadi/BIRSBIO2020.scNMTseq.PLS/commit/d60858fe3b458431422407af83319ea4910ab6b0). However, dockerbuild-push-action timed out once and I had to re-run the workflow (it hadn't hit the GHA timeout so I'm assuming there was something with the dockerbuild action). If so, is there a way to increase the runtime limit?
